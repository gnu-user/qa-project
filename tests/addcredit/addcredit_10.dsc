TITLE
    Non-Privileged Mode Prompt
DESCRIPTION
    Run the login transaction using any non-admin account. Initiate the 
    addcredit transaction. Verify that the user is prompted to enter the amount 
    of credit to add.
DEPENDENCIES
