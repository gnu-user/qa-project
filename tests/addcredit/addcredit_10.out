Welcome.
Current user accounts file read successfully.
Available tickets file read successfully.
> 
Enter username: 
Username accepted.
> 
Enter the amount of credit to add: 
> 